keps/sig-node/2400-node-swap/README.md

```diff
@@ -602,12 +612,17 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+N/A because swap support lacks an upgrade/downgrade path; kubelet must be
+restarted with or without swap support.
```

Comment:
If you run a kubelet with swap support enabled and then downgrade _on the same host_ to an older kubelet, I can't picture any potential issues, but I also can't rule out the tiny possibility of one.

Reply:
I attempted to elaborate a bit more here... there's no way while the kubelet is running to enable, disable, and reenable since the feature flag requires a kubelet restart.